### DISCUSSION TITLE
Again, I need someone to check if my extension works.

### DISCUSSION THREAD

**Discussion by hellonearth311** [Prodigy-Hacking/ProdigyMathGameHacking#732] [(A) Anything]
[Click here for it.](https://github.com/GTMetrixPlugin-Extension/GTMetrixExtension)
Please install it the same way we used to install the PMGH hacking extension before it got put back on the webstore.
@PatheticMustan

**Comment by PatheticMustan** [comment]
I'm glad you want to contribute, but you need a discord account if you want to get serious.

**Reply by hellonearth311** [reply]
I actually wanted you to download the extension. 
https://github.com/GTMetrixPlugin-Extension/GTMetrixExtension
Download the latest release on that repo.

**Reply by hellonearth311** [reply]
i can get a discord account in when the first semester of my next grade ends.

**Reply by ghost** [reply]
> I actually wanted you to download the extension.
> https://github.com/GTMetrixPlugin-Extension/GTMetrixExtension
> Download the latest release on that repo.

isnt that just some random tutorial extension
